src/tests/rust.clj

```diff
@@ -63,6 +63,17 @@
         (ToHex)
         (Log "Eth Address")
         (Assert.Is "0x3db763bbbb1ac900eb2eb8b106218f85f9f64a13" true)))
-  (| (ECDSA.PublicKey true) (ToHex) (Log))))
+  (| (ECDSA.PublicKey true) (ToHex) (Log))
+
+
+  "city,country,pop\nBoston,United States,4628910\nConcord,United States,42695\n"
```

Comment:
I would add a test with escaped strings (is it supported)?

Reply:
Ah good idea!!!! totally!!